metrics/src/main/java/org/wildfly/extension/metrics/_private/MetricsLogger.java

```diff
@@ -52,4 +52,8 @@ public interface MetricsLogger extends BasicLogger {
     @LogMessage(level = ERROR)
     @Message(id = 5, value = "Malformed name.")
     void malformedName(@Cause Exception exception);
+
+    @LogMessage(level = INFO)
+    @Message(id = 6, value = "The system may have more than one metrics gathering system enabled. Please see the administration guide for more information.")
```

Comment:
Lets be more specific here, i.e. "multiple metrics collection systems"

Reply:
Updated